sdk/Trace/Tracer.php

```diff
@@ -123,16 +127,17 @@ public function setActiveSpan(API\Span $span): void
     }
 
     /**
-     * @param string $name
```

Comment:
Is there a reason you removed these?

Reply:
To me they're pointless given the related arguments already have a PHP type hint.